ta/Makefile

```diff
@@ -5,6 +5,8 @@ ifeq ($O,)
 $(error output path should be specified when calling this makefile)
 endif
 
+LDFLAGS=
```

Comment:
But this would again not work in yocto environment as LDFLAGS is set by the environment so it will override LDFLAGS for TA too. Based on my understanding why do we need to configure LDFLAGS by external environment to compile trusted application as its headers and libraries would be provided by Trusted OS itself? For client application it seems right to override LDFLAGS. Kindly correct me if I am missing something.